prowler/providers/aws/services/rds/rds_instance_deprecated_engine_version/rds_instance_deprecated_engine_version.py

```diff
@@ -0,0 +1,30 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.rds.rds_client import rds_client
+
+
+class rds_instance_deprecated_engine_version(Check):
+    def execute(self):
+        findings = []
+        available_versions = []
+
+        for instance in rds_client.db_instances:
+            available_versions.clear()
```

Comment:
Why do you need to clear this `available_versions` variable? If this variable is only used within the `for` loop scope it'd better to define it there, so moving line 8 to line 11.

Reply:
Done!